openmetadata-service/src/main/java/org/openmetadata/service/search/SearchEventPublisher.java

```diff
@@ -68,7 +67,7 @@ public void onShutdown() {
   }
 
   public static void updateElasticSearchFailureStatus(
-      EntityInterface entity, EventPublisherJob.Status status, String failureMessage) {
+      Object entity, EventPublisherJob.Status status, String failureMessage) {
```

Comment:
From what I see in that method we are simply doing `entity.toString()` so we should be able to accept any kind of objects